lua/telescope/builtin/files.lua

```diff
@@ -496,7 +496,6 @@ files.current_buffer_tags = function(opts)
   return files.tags(vim.tbl_extend("force", {
     prompt_title = 'Current Buffer Tags',
     only_current_file = true,
-    hide_filename = true,
```

Comment:
I think it still makes sense to hide the filename for `current_buffer_tags`. I don't know the new interface yet but you update that part?